src/bumping/npm.ts

```diff
@@ -21,17 +21,36 @@ const getPackageVersion = async (packageMD: PackageMetadata) => {
   }
 };
 
+const isPackageJSONVersionHigher = (packageJSONVersion: string, bumpedVersion: string) => {
+  const semverMarkersPackageJSON = packageJSONVersion.split(".");
+  const semverMarkersBumped = bumpedVersion.split(".");
+  for (let i = 0; i < 3; i++) {
+    if (Number(semverMarkersPackageJSON[i]) > Number(semverMarkersBumped[i])) {
+      return true;
+    }
+  }
+
+  return false;
+}
```

Comment:
hah, this is a nice way to do it - won't handle build metadata but I'm OK with that crashing for someone and them adding that